src/SearchParameters/index.js

```diff
@@ -960,6 +1009,60 @@ SearchParameters.prototype = {
     return this.setQueryParameters(modification);
   },
   /**
+   * Remove a facet from the facets attribute of the helper configuration, if it
+   * is present.
+   * @method
+   * @param {string} facet facet name to remove
+   * @return {SearchParameters}
+   */
+  removeFacet: function removeFacet(facet) {
+    if (!this.isConjunctiveFacet(facet)) {
+      return this;
+    }
+
+    const newFacets = this.facets.slice();
+    newFacets.splice(newFacets.indexOf(facet), 1);
```

Comment:
I'm used to avoiding using filter as a remove function, but filter does make for cleaner code. 👍